addons/purchase_stock/tests/test_average_price.py

```diff
@@ -130,3 +130,52 @@ def test_00_average_price(self):
         self.assertEqual(product_cable_management_box.qty_available, 20.5, 'Reception of purchase order in grams leads to wrong quantity in stock')
         self.assertEqual(round(product_cable_management_box.standard_price, 2), 78.05,
             'Standard price as average price of third reception with other UoM incorrect! Got %s instead of 78.05' % (round(product_cable_management_box.standard_price, 2)))
+
+    def test_inventory_user_svl_access(self):
+        """ Test to check if Inventory/User is able to validate a
+        transfer when the product has been invoiced already """
+
+        user = self.env['res.users'].create({
+            'name': 'Inventory Manager',
+            'login': 'test',
+            'email': 'test@test.com',
+            'groups_id': [(6, 0, [self.env.ref('stock.group_stock_user').id])]
+        })
+
+        avco_product = self.env['product.product'].create({
+            'name': 'Average Ice Cream',
+            'type': 'product',
+            'categ_id': self.stock_account_product_categ.id,
+        })
+        avco_product.categ_id.property_cost_method = 'average'
+        avco_product.standard_price = 300
+
+        # Add a previous move to triger "run_fifo_vaccum"
+        out_move = self.env['stock.move'].create({
+            'name': 'out move',
+            'product_id': avco_product.id,
+            'location_id': avco_product.property_stock_production.id,
```

Comment:
you won't create any layer with this pair of locations. you should make a move from Stock -> Cust or Stock -> prod